src/session.rs

```diff
@@ -254,6 +254,14 @@ impl Session {
                     // then reply code 503 (command invalid).
                     let chans = channels.lock().unwrap();
                     let chan_id = frame.channel;
+
+                    if chan_id == 0 {
+                        info!(
+                            "Dropping frame to channel 0: {:?}",
+                            String::from_utf8_lossy(frame.payload.inner())
```

Comment:
Are we trying to parse the payload as UTF8, because it is expected to contain an error message string?

Reply:
That is what I was hoping for. Unfortunately I'm going to close this patch -- it is not good to see these lines of output during startup when the library _is_ handling these messages.

Would it be acceptable to you if I started a thread to listen on channel 0 and log those as dropped? I could possibly combine that with my patch to add heartbeat support :)